### PR TITLE
Issue #SB-16489 feat: Hiding Download and Share for Limited Sharing Content

### PR DESCRIPTION
**Pull request by NavKumarV**

### CODE REVIEW

File: src/app/content-details/content-details.page.ts [hunk 1]
```diff
@@ -207,6 +209,8 @@ export class ContentDetailsPage implements OnInit, OnDestroy {
       this.resumedCourseCardData = extras.resumedCourseCardData;
       this.isSingleContent = extras.isSingleContent;
       this.resultLength = extras.resultsSize;
+      this.hideDownloadShare = (this.cardData.contentData &&
```
Comment (AmiableAnil) [inline src/app/content-details/content-details.page.ts]: Add this in extractApiResponse() as well.

File: src/app/content-details/content-details.page.ts [hunk 2]
```diff
@@ -791,6 +800,20 @@ export class ContentDetailsPage implements OnInit, OnDestroy {
       });
   }
 
+  checkLoginForLimitedSharing(switchUserFlag) {
```
Comment (AmiableAnil) [inline src/app/content-details/content-details.page.ts]: rename it to handleContentPlay

Comment (NavKumarV) [reply src/app/content-details/content-details.page.ts]: Changed to handleContentPlay

File: src/app/content-details/content-details.page.ts (hunk 1 above)
Comment (NavKumarV) [reply src/app/content-details/content-details.page.ts]: This changes are done

**Pull request merged**